### PR TITLE
Show retry on bootstrap failure

### PR DESCRIPTION
**Pull request by gabriel**
Tested by blocking api.keybase.io lots (and unblocking before reload).

2016-09-23 at 3 03 pm

### CODE REVIEW

**Comment by gabriel** [comment]
@keybase/react-hackers

**Comment by keybase-ci-visdiff** [comment]
:mag_right: The commits 3ae1b5b14a55...8bf1835fb520 introduced some visual changes on linux:
 * Changed: **Intro-User_who_tried_to_login_from_revoked_device** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-User_who_tried_to_login_from_revoked_device-3ae1b5b14a55.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-User_who_tried_to_login_from_revoked_device-8bf1835fb520.png)  
   Intro-User_who_tried_to_login_from_revoked_device comparison
 * Changed: **Intro-User_who_just_revoked_device** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-User_who_just_revoked_device-3ae1b5b14a55.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-User_who_just_revoked_device-8bf1835fb520.png)  
   Intro-User_who_just_revoked_device comparison
 * Changed: **Intro-First_time_user** [(before)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-First_time_user-3ae1b5b14a55.png) [(after)](https://keybase-app-visdiff.s3.amazonaws.com/1474669040477-3ae1b5b14a55-8bf1835fb520-linux/Intro-First_time_user-8bf1835fb520.png)  
   Intro-First_time_user comparison

**Comment by chromakode** [comment]
@gabriel see the visdiffs here -- looks like the dumb sheet needs to be updated to reflect the changes you made here.

**Comment by gabriel** [comment]
k... fixed visdiffs and added one for boot failure

**Comment by keybase-ci-visdiff** [comment]
The commits fbf0d7d35acc...24a8e6918cc7 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Intro-Failure**  
   Intro-Failure rendered
</summary>

**Comment by keybase-ci-visdiff** [comment]
The commits f15fde549dfc...55cc6938f583 introduce visual changes on linux. <details><summary>:mag_right: 1 new</summary>

 * Added: **Intro-Failure**  
   Intro-Failure rendered
</summary>

File: shared/login/forms/intro.render.desktop.js [hunk 1]
```diff
@@ -63,6 +85,12 @@ const stylesHeader = {
 const stylesHeaderSub = {
   marginTop: 3,
 }
+const stylesMessage = {
+  marginLeft: 40,
```
Comment (cjb) [inline shared/login/forms/intro.render.desktop.js]: @gabriel if the number 40 isn't important, perhaps use a globalMargins instead?

**Pull request merged**